tests/testthat/test-FunctionReporter.R

```diff
@@ -90,7 +90,6 @@ test_that('FunctionReporter Methods Work', {
                  , info = "derived edges do not match expected edges"
                  )
 
-    # TODO: Need to test that nodes were properly extracted
```

Comment:
created #180, @bburns632 feel free to edit the description on there to elaborate, I don't think what I've written so far is sufficient for a not-us developer to pick it up

Reply:
Should also be covered by https://github.com/UptakeOpenSource/pkgnet/blob/master/tests/testthat/test-NetworkMeasures.R